src/homeScreen.js

```diff
@@ -1,33 +1,90 @@
 const match = require('./match');
-const { Label, HomeLabel } = require('./labels');
+const text = require('./text');
 
 class HomeScreen {
+    /**
+     * @example home.ActiveCarbohydratesLabel().tap();
+     */
+    ActiveCarbohydratesLabel() {
```

Comment:
💯 I like your best practice of the suffix indicating what kind of thing it is (e.g. `Label`, `Button`).

Reply:
cheers